### ISSUE TITLE
Improve error message on missing VCS tool

### ISSUE THREAD

**Issue by timoreimann** [Masterminds/glide#426]
When trying to `glide install` on my project, I got the following error message:

```
[WARN] Unable to checkout bitbucket.org/ww/goautoneg
[ERROR] Update failed for bitbucket.org/ww/goautoneg: Unable to get repository
```

@sdboyer [helped me](https://gitter.im/Masterminds/glide?at=573ba2f2c61823687d3cd692) realize that I was just missing `hg` on my machine. The error message should be more clear on this circumstance.

**Comment by mattfarina** [comment]
@timoreimann Thanks for the report. I've been wondering when something like this would happen. I've added an issue to the VCS project where this should live.

PRs are welcome or I'll get to it soon-ish.

**Comment by timoreimann** [comment]
This was fixed through #43, so closing.

**Comment by timoreimann** [comment]
Meant to ref Masterminds/vcs#43 -- sorry for the link pollution!
